sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/PartitionKeyBuilder.java

```diff
@@ -0,0 +1,90 @@
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Undefined;
+import com.azure.cosmos.implementation.routing.PartitionKeyInternal;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PartitionKeyBuilder {
+    private List<Object> partitionKeyValues;
+
+    /**
+     * Constructor. CREATE a new instance of the PartitionKeyBuilder object.
+     */
+    public PartitionKeyBuilder()
+    {
```

Comment:
nit: java code style '{' on the same line as method, for other new methods too.

Reply:
Fixed